x-pack/plugins/security_solution/public/entity_analytics/translations.ts

```diff
@@ -130,3 +130,117 @@ export const PREVIEW_QUERY_ERROR_TITLE = i18n.translate(
     defaultMessage: 'Invalid query',
   }
 );
+
+export const UPDATE_AVAILABLE = i18n.translate('xpack.securitySolution.riskScore.updateAvailable', {
+  defaultMessage: 'Update available',
+});
+
+export const START_UPDATE = i18n.translate('xpack.securitySolution.riskScore.startUpdate', {
+  defaultMessage: 'Start update',
+});
+
+export const UPDATING_RISK_ENGINE = i18n.translate(
+  'xpack.securitySolution.riskScore.updatingRiskEngine',
+  {
+    defaultMessage: 'Updating risk engine...',
+  }
+);
+
+export const UPDATE_RISK_ENGINE_MODAL_TITLE = i18n.translate(
+  'xpack.securitySolution.riskScore.updateRiskEngineModa.title',
+  {
+    defaultMessage: 'Do you want to update the entity risk engine?',
+  }
+);
+
+export const UPDATE_RISK_ENGINE_MODAL_EXISTING_USER_HOST_1 = i18n.translate(
+  'xpack.securitySolution.riskScore.updateRiskEngineModal.existingUserHost_1',
+  {
+    defaultMessage: 'Existing user and host risk score transforms will be deleted',
+  }
+);
+
+export const UPDATE_RISK_ENGINE_MODAL_EXISTING_USER_HOST_2 = i18n.translate(
+  'xpack.securitySolution.riskScore.updateRiskEngineModal.existingUserHost_2',
+  {
+    defaultMessage: ', they are no longer required.',
```

Comment:
@rylnd I see that the [UX copy](https://www.figma.com/file/jwzKiq2IyxapRlAzmz1kyJ/8.9-%7C-6546-%7C-Migrate-to-new-scoring-engine-%7C-Entity-Analytics?type=design&node-id=143-84665&mode=design&t=h99GdItjePQ6bRFZ-0) specifies the reason as: 

`Existing user and host risk score transforms will be deleted, they are not required for the new entity risk engine.`

IMO, it reads adequately. cc: @craig-abbott in case I missed anything.

Reply:
@rylnd @SourinPaul I'm happy to go with the more concise "as they are no longer required."

In terms of explaining why they are no longer required, I'd tried to allude to that by saying they're not needed for the new risk engine, but it doesn't really go into detail, hence I guess you wanting to make it more concise haha. 

There is a link to "learn more about updating the new risk engine" where I assumed we could provide more technical detail. But I'm making assumptions that documentation would be published I could link to. I'm happy to re-look at it in the modal itself if necessary?